src/Symfony/Component/Console/Helper/TableStyle.php

```diff
@@ -99,49 +161,75 @@ public function getHorizontalBorderChar()
      * @param string $verticalBorderChar
      *
      * @return $this
+     *
+     * @deprecated since Symfony 4.1. Use {@link setVerticalBorderChars()} instead.
      */
     public function setVerticalBorderChar($verticalBorderChar)
     {
-        $this->verticalBorderChar = $verticalBorderChar;
+        @trigger_error(sprintf('Method %s() is deprecated since Symfony 4.1. Use setVerticalBorderChars() instead.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
Deprecating a non-internal method in favor of an internal one looks weird to me.

Reply:
This one isn't internal. Are you talking about `getBorderChars()` maybe?

Reply:
`since Symfony 4.1, use set...`

Reply:
Fixed